code/modules/hydroponics/grown/replicapod.dm

```diff
@@ -205,10 +205,11 @@
 	podman.set_cloned_appearance()
 
 	//Get the most plentiful reagent, if there's none: get water
-	var/most_plentiful_reagent = list(/datum/reagent/water = 0)
+	var/list/most_plentiful_reagent = list(/datum/reagent/water = 0)
 	for(var/reagent in reagents_add)
 		if(reagents_add[reagent] > most_plentiful_reagent[most_plentiful_reagent[1]])
-			most_plentiful_reagent = list("[reagent]" = reagents_add[reagent])
+			most_plentiful_reagent.Cut()
+			most_plentiful_reagent[reagent] += reagents_add[reagent]
```

Comment:
you sure this shouldnt be just = ?

Reply:
I don't think it really matters, I just used the same syntax as LAZYADDASSOC
(and yes it does work I tested the final product this time)

Reply:
If you prefer I do it differently just tell me, I don't really care

Reply:
i think = is easier to understand

Reply:
done